source/stylesheets/_refills-styles.scss

```diff
@@ -374,84 +374,109 @@ pre[class*="language-"] {
   line-height: 1em;
 }
 
-.snippets-table {
+.snippets {
   display: none;
 
-  td {
-    border-bottom: 0;
-  }
+  .snippets-table {
+    td {
+      border-bottom: 0;
+    }
 
-  .snippet:nth-child(4n) {
-    display: none;
+    .snippet-javascript:nth-child(4) {
+      display: none;
+    }
+
+    .snippet-scss:last-child {
+      display: none;
+    }
   }
 }
 
+.copy-variables {
+  @include linear-gradient(white, whitesmoke);
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color `whitesmoke` should be written in hexadecimal form as `#f5f5f5`